### PR TITLE
Add BumbleBee carving artifact

### PR DESCRIPTION
**Pull request by angelovioletti**
Hello,
I've created this carving artifact to detect BumbleBee processes running on compromised systems and subsequently extract the command and control servers from the process memory.

### CODE REVIEW

File: artifacts/definitions/Windows/Carving/BumbleBee.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            $IP
+        }
+
+sources:
+  - precondition:
+      SELECT OS From info() where OS = 'windows'
+
+    query: |
+        -- Find velociraptor process
+        LET me = SELECT Pid
```
Comment (scudette) [inline artifacts/definitions/Windows/Carving/BumbleBee.yaml:50]: This is kind of pointless no? you can get the pid with `LET me <= getpid()`

File: artifacts/definitions/Windows/Carving/BumbleBee.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                  FROM pslist(pid=getpid())
+
+        -- Find all processes and add filters
+        LET processes = SELECT Name AS ProcessName, CommandLine, Pid
+                        FROM pslist()
+                        WHERE Name =~ ProcessRegex
+                            AND format(format="%d", args=Pid) =~ PidRegex
```
Comment (scudette) [inline artifacts/definitions/Windows/Carving/BumbleBee.yaml:57]: You dont actually need this any more - the regex match now supports integers as well  `AND Pid =~ PidRegex`

**Comment by scudette** [comment]
Thanks for this PR

Do you think this is better in the artifact exchange? We try to keep only more generic artifacts that have much wider use than a single malware in the built in set.

**Review by angelovioletti — COMMENTED** [review]
Improvement on the artifact.

**Pull request closed without merging**